data/json/itemgroups/SUS/clothes_store.json

```
@@ -69,7 +69,8 @@
               { "item": "needle_curved", "prob": 75 },
               { "item": "scissors", "prob": 90 },
               { "item": "knitting_needles", "prob": 20 },
-              { "item": "sewing_kit", "prob": 85, "charges-min": 0 }
+              { "item": "sewing_kit", "prob": 85, "charges-min": 0 },
+              { "item": "kevlar_shears", "prob": 75 }
```

```suggestion
              { "item": "kevlar_shears", "prob": 5 }
```
Again, this is a place you might find them, but not on anything resembling the frequency you'd find a sewing kit. Even this is exceptionally high.